sycl/test/basic_tests/buffer/buffer_full_copy.cpp

```diff
@@ -16,7 +16,7 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
 //===----------------------------------------------------------------------===//
-
+#include "../../helpers.hpp"
 #include <CL/sycl.hpp>
 #include <cassert>
```

Comment:
```suggestion

```
Do we still need this include?

Reply:
It shouldn’t be necessary - I will fix this on Monday.